perfTest/run.sh

```diff
@@ -1,24 +1,28 @@
 #!/usr/bin/env bash
 set -e
 
+# run in this script's parent directory
+cd "${0%/*}"
+
 export NO_LOG_SUCCESS=1
 
-# Reset the database
-dropdb graphile_worker_perftest || true;
-createdb graphile_worker_perftest;
+# if connection string not provided, assume postgres is available locally
+export PERF_DATABASE_URL=${TEST_CONNECTION_STRING-graphile_worker_perftest}
+
+# drop and recreate the test database
+node ./recreateDb.js
 
 # Install the schema
-DATABASE_URL="graphile_worker_perftest" node ../dist/cli.js --once
+DATABASE_URL=$PERF_DATABASE_URL node ../dist/cli.js --once
```

Comment:
Should always use quotes to avoid shell expansion

```suggestion
DATABASE_URL="$PERF_DATABASE_URL" node ../dist/cli.js --once
```